cmd/restic/cmd_tag.go

```diff
@@ -84,19 +90,21 @@ func changeTags(ctx context.Context, repo *repository.Repository, sn *restic.Sna
 		// Save the new snapshot.
 		id, err := restic.SaveSnapshot(ctx, repo, sn)
 		if err != nil {
-			return false, err
+			return nil, false, err
 		}
 
 		debug.Log("new snapshot saved as %v", id)
 
 		// Remove the old snapshot.
 		if err = repo.RemoveUnpacked(ctx, restic.SnapshotFile, *sn.ID()); err != nil {
-			return false, err
+			return nil, false, err
 		}
 
 		debug.Log("old snapshot %v removed", sn.ID())
+
+		return &changedSnapshot{Original: *sn.ID(), New: id}, changed, nil
```

Comment:
The information about changed snapshots should be printed right when the snapshot was modified. Otherwise, the output will be lost if the `tag` command is canceled.

Reply:
Done.